### ISSUE TITLE
VC with status

### ISSUE THREAD

**Issue by ssi-vc-dev** [Sphereon-Opensource/did-siop-auth-web-demo#98]
Hey guys, I was using your demo and both OID4VCI and OID4VP work perfectly! Good job. 

I have one question about issuing VC with status / credentialStatus. 
From what I can see, currently, the credentials issued within the demo don't have status.
I want to have something like this https://www.npmjs.com/package/@sphereon/vc-status-list
Which plugins i should add to the agent in this demo, to be able to issue VC with status?

**Comment by nklomp** [comment]
Yes in our SSI-SDK we actually have plugins that both can handle the management side of things for revocations, as well as automatic random index assignments in a list when issuing a credential.
See: https://github.com/Sphereon-Opensource/SSI-SDK/tree/develop/packages/vc-status-list-issuer-rest-api and https://github.com/Sphereon-Opensource/SSI-SDK/tree/develop/packages/vc-status-list-issuer-drivers

Unfortunately we do not have documentation for it yet

There is support on the RP or Wallet side. Whenever a StatusList2021 is encountered it will check the status.

Reason we didn't include it is because it adds complexity to an already pretty complex demo to setup.

**Comment by ssi-vc-dev** [comment]
Thanks! 
I'm trying to set up the plugins with the demo. 
Do you maybe have an idea, is it sufficient if I add them within the packages/agent? Or they should be added in the packages/oid4vci-demo-frontend && /packages/oid4vp-demo-frontend too?

**Comment by nklomp** [comment]
You will need to add them in the demo as well, as there currently is no automatic recognition of the status-list issuer being present from the agent itself. We still need to develop that part. Of course then you would be able to configure on the agent itself which statusList should be associated with which credential type(s) and then it would include them automatically. Now you will have to create a statusList entry (first create an actual statuslist) using the methods, and then include them in the Credential Payload.

**Comment by ChSonendra** [comment]
> Thanks! I'm trying to set up the plugins with the demo. Do you maybe have an idea, is it sufficient if I add them within the packages/agent? Or they should be added in the packages/oid4vci-demo-frontend && /packages/oid4vp-demo-frontend too?

hey bro did you had any luck on revocation??

**Comment by nklomp** [comment]
See
https://ssisdk.docs.sphereon.com/status-list/introduction
and
https://ssisdk.docs.sphereon.com/exercises/status-list/status-list-exercises
